Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-subitemlistview', function (Y) {
+    "use strict";
+    /**
+     * Provides the subitem list view.
+     *
+     * @module ez-subitemlistview
+     */
+    Y.namespace('eZ');
+
+    var IS_PAGE_LOADING = 'is-page-loading',
+        IS_DISABLED = 'is-disabled';
+
+    function linkIsDisabled(link) {
```

Comment:
it's not exposed in any way (really private) and I guess the name is clear enough